editor/plugins/animation_library_editor.cpp

```diff
@@ -621,6 +621,9 @@ void AnimationLibraryEditor::update_tree() {
 		return;
 	}
 
+	new_library_button->set_icon(get_editor_theme_icon(SNAME("Add")));
+	load_library_button->set_icon(get_editor_theme_icon(SNAME("Load")));
```

Comment:
This should be in `NOTIFICATION_THEME_CHANGED`.

Reply:
Alright, will look into it later today!^^

Reply:
Done!